python/src/deltachat/account.py

```diff
@@ -618,8 +618,6 @@ def run_account(self, addr=None, password=None, account_plugins=None, show_ffi=F
             assert addr and password, "you must specify email and password once to configure this database/account"
             self.set_config("addr", addr)
             self.set_config("mail_pw", password)
-            self.set_config("mvbox_move", "0")
-            self.set_config("sentbox_watch", "0")
```

Comment:
mvbox makes some sense due to mailcow issue, but `sentbox_watch` doesn't makes much sense since it is meant for usage with other MUA with is not applicable for a bot

Reply:
Then turning it off here explicitly does not really make sense? And we can merge it as it is?

Reply:
ah yes, makes sense to delete then, it seems this is from the time it was on by default